app/models/cmr.rb

```diff
@@ -53,7 +53,11 @@ def self.get_raw_concept(concept_id, revision_id = nil)
   # If Cmr revision id isn't the latest, we pull the latest and ingest it.
 
   def self.update_collections(current_user)
-    records = Record.where state: [Record::STATE_OPEN.to_s, Record::STATE_IN_ARC_REVIEW.to_s, Record::STATE_READY_FOR_DAAC_REVIEW.to_s], recordable_type: "Collection"
+    if current_user.mdq_curator?
+      records = Record.all_records(:mdq_mode).where state: [Record::STATE_OPEN.to_s, Record::STATE_IN_ARC_REVIEW.to_s, Record::STATE_READY_FOR_DAAC_REVIEW.to_s], recordable_type: "Collection"
```

Comment:
what if instead of thinking of this as a user mode, what if it was a filter mode. There could be any number of actions in the future that changes how you're filtered. Maybe the user will say they only want to view one provider at a time or something? Then I am suggesting that when you login your role gives you a default filter rule. Today this may never change, but it could.

Reply:
I agree, it should be the application_mode not the specific mode.   This actually might be a bug, need to look closer.  good catch.

Reply:
This isn't a bug the Record.all_records takes an application mode as an argument, it just happens to be we determine the mode is based on the user's role, but we can change the application_mode to whatever we want in the future.